man/mkdocs/mkdocs.yml

```diff
@@ -26,6 +26,8 @@ theme:
     - search.suggest
     - search.highlight
     - content.code.copy
+    - content.tabs.link
```

Comment:
Can you please separate the tabs into a new PR to get it merged quickly, including the linked tabs option here? I was able to enable the tabs (locally), but the doc (mis)lead me to include some custom JS to keep the tabs in sync and that was not working for me.

Reply:
https://github.com/OSGeo/grass/pull/5277